src/app/components/player/ExtraOptions.tsx

```diff
@@ -94,6 +95,25 @@ const ExtraOptions: React.FC = observer(() => {
             </>
           )}
         />
+        <Toggle
+          checked={player.buffs.soulflameHorn}
+          setChecked={(c) => store.updatePlayer({ buffs: { soulflameHorn: c } })}
+          label={(
+            <>
+              <img src={soulflame.src} width={18} className="inline-block" alt="" />
+              {' '}
+              Buffed by Soulflame horn
```

Comment:
For consistency with other things in the UI I would change this to say either "Entice buff" or "Soulflame horn buff"